AWS/legos/aws_get_ebs_volume_for_low_usage/aws_get_ebs_volume_for_low_usage.py

```diff
@@ -54,10 +54,11 @@ def aws_get_ebs_volume_for_low_usage(handle, region: str = "", threshold_days: i
             for volume in response:
                 ebs_volume = {}
                 volume_id = volume["VolumeId"]
+                volume_size = volume['Size']
                 cloudwatch = handle.client('cloudwatch', region_name=reg)
-                cloudwatch_response = cloudwatch.get_metric_statistics(
+                read_metric_data = cloudwatch.get_metric_statistics(
                                     Namespace='AWS/EBS',
-                                    MetricName='VolumeUsage',
+                                    MetricName='VolumeReadBytes',

```

Comment:
According to documentation- VolumeReadBytes provides information on the read operations in a specified period of time. There is VolumeWriteBytes too...

Reply:
we already implemented and compare both read operations and write operations.